### PR TITLE
fix: include <ctime>

### PR DESCRIPTION
**Pull request by Solonix-Chu**

### CODE REVIEW

**Comment by Solonix-Chu** [comment]
I have read the CLA Document and I hereby sign the CLA.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,16 @@
+clucene-core (2.3.3.4-deepin1) unstable;urgency=medium
+
+  *Fix build on RISC-V
+
+ --Z8MAN8 <1468559561@qq.com> Mon,28 Nov 2022 22:44:20 +0800
+
+clucene-core (2.3.3.4+dfsg-1.1) UNRELEASED; urgency=medium
```
Comment (RevySR) [inline debian/changelog:7]: 需要移除垃圾提交 版本号是2.3.3.4.1-deepin1

File: debian/patches/series [hunk 2]
```diff
@@ -5,3 +5,4 @@ multiarch.patch
 bug715474-no-boost-headers.patch
 bug718187-ftbfs-hurd.patch
 gcc-version.diff
+add-missing-lib.patch
```
Comment (RevySR) [inline debian/patches/series:8]: 补丁名字最好是fix-gcc12-build.patch

File: debian/changelog (hunk 1 above)
Comment (Solonix-Chu) [reply debian/changelog:7]: 已修改

File: debian/patches/series (hunk 2 above)
Comment (Solonix-Chu) [reply debian/patches/series:8]: 好的，下次注意

**Pull request closed without merging**